src/components/comment-list-item.component.js

```diff
@@ -226,6 +226,13 @@ export class CommentListItem extends Component {
             {defaultRenderer(node.children, parent)}
           </Text>
         );
+      } else if (node.name === 'li' && parent.name === 'ol') {
+        return (
+          <Text key={index} style={textStyle}>
+            {`${Math.ceil((index + 1) / 2)}.`}{' '}
```

Comment:
Oh yes of course ! I didn't put this `{' '}`, it was probably added by the pre-commit hook.

I push an update now